cms/settings/local.py.example

```diff
@@ -1,8 +1,10 @@
-CACHES = {
-    'default': {
-        'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
-    }
-}
+# uncomment the following lines if running outside of the Docker container
+# --- outside Docker ---
+# import dj_database_url
+
+# DATABASES = {"default": dj_database_url.config(default="sqlite:///cms.sqlite3")}
```

Comment:
Whilst it's tested, SQLite could be have very differently, especially if we have any postgres-specific implementations (unlikely, but you never know). I suggest we just enforce postgres.